[#64] refactoring of activemq artemis (#65)
arkmq-org/activemq-artemis-broker-init-image: README.md

# ArkMQ Broker Init container image powered by [Apache Artemis](https://activemq.apache.org/components/artemis/)

> ArkMQ Broker Init is a container image for configuring [Apache Artemis](https://activemq.apache.org/components/artemis/) broker instances in containerized environments.

Reviewer: ```suggestion
# ArkMQ Broker Init container image powered by [Apache Artemis](https://artemis.apache.org/)
ArkMQ Broker Init is a container image for configuring [Apache Artemis](https://artemis.apache.org/) broker instances in containerized environments.
```

Reviewer: Never mind you already updated them.

# Repository

> The ArkMQ Broker container images are pushed to https://quay.io/repository/arkmq-org/arkmq-org-broker-init

Reviewer: ```suggestion
The ArkMQ Broker Init container images are pushed to https://quay.io/repository/arkmq-org/arkmq-org-broker-init
```

# Tags

> The ArkMQ Broker container images have release tags and special tags.

Reviewer: ```suggestion
The ArkMQ Broker Init container images have release tags and special tags.
```
